pytest_playwright/pytest_playwright.py

```diff
@@ -411,3 +60,17 @@ def pytest_addoption(parser: Any) -> None:
         default=False,
         help="Whether to take a full page screenshot",
     )
+
+def pytest_addoption(parser: pytest.Parser) -> None:
+    parser.addini("playwright_pytest_asyncio", "Use asyncio Playwright fixtures. Default: when 'pytest-asyncio' plugin is activated=True, otherwise False", type="bool", default=None)
+
+
+def pytest_configure(config: pytest.Config) -> None:
+    if (config.getini("playwright_pytest_asyncio") is None and config.pluginmanager.hasplugin("asyncio")) or config.getini("playwright_pytest_asyncio"):
```

Comment:
To share, I have seen others using AnyIO for `async` testing: https://anyio.readthedocs.io/en/1.4.0/testing.html

It might be good to account for other `async` plugins that can trigger registration of `PytestPlaywrightAsyncio`

Reply:
We'll do it on a global level for now since it seems simpler. Sync and Async fixtures seem to be significant different, hence deciding this on a test-level won't work. Browser and Playwright instance in the Pytest plugin are e.g. on session scope, so that would also end up in clashes / conflicts.